addons/event_booth/models/event_event.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo import Command
+
+
+class Event(models.Model):
+    _inherit = 'event.event'
+
+    event_booth_ids = fields.One2many(
+        'event.booth', 'event_id', string='Booths', copy=True,
+        compute='_compute_event_booth_ids', readonly=False, store=True)
+    event_booth_count = fields.Integer(
+        string='Total Booths',
+        compute='_compute_event_booth_count')
+    event_booth_count_available = fields.Integer(
+        string='Available Booths',
+        compute='_compute_event_booth_count')
+    event_booth_category_ids = fields.Many2many(
+        'event.booth.category', compute='_compute_event_booth_category_ids')
+    event_booth_category_available_ids = fields.Many2many(
```

Comment:
so those are the category for the event booth that are still available, right ? Add help could be cool
and add why it is needed